pkg/ingester/ingester.go

```diff
@@ -64,7 +64,7 @@ type Config struct {
 	ingesterClientFactory func(cfg client.Config, addr string) (client.HealthAndIngesterClient, error)
 
 	QueryStore                  bool          `yaml:"-"`
```

Comment:
we can maybe get rid of this because now we mostly rely on `QueryStoreMaxLookBackPeriod` to enable/disable querying the store from ingesters.

Reply:
It seems correct what @sandeepsukhani is saying ?